### ISSUE TITLE
Homepage - Improve Focus support

### ISSUE THREAD

**Issue by perclasson** [flutter/gallery#36]
- On the homepage you should get focus on the settings icon first, then studies, then the demos.
- Settings should be settable with Focus (?).

Migrated from https://github.com/material-components/material-components-flutter-gallery/issues/569

**Comment by perclasson** [comment]
This seems to be done already. No change needed here. I assume there have been changes to the flutter/flutter support for this.

**Comment by guidezpl** [comment]
Working on this atm since Focus + Semantics are somewhat coupled. Remaining work will be in #73
